src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -760,6 +760,11 @@ internal IEnumerable<NodeSearchElement> GetMatchingSearchElements()
             else if (PortViewModel.PortModel.PortType == PortType.Output)
             {
                 portType = PortViewModel.PortModel.GetOutPortType();
+                //if the custom node output name contains spaces, try using the first word.
+                if (PortViewModel.PortModel.Owner is Graph.Nodes.CustomNodes.Function && portType.Contains(" "))
```

Comment:
Should we generalize this check to include any white space?

Reply:
I can do that- I don't think any whitespace character is valid in a DS type name.